src/django/api/views/v1/opensearch_query_builder/opensearch_query_builder.py

```diff
@@ -66,9 +51,12 @@ def __build_number_of_workers(self, field, range_query):
             }
         })
 
-    def __build_country(self, field):
+    def _build_country(self, field):
```

Comment:
> Moreover, it should be marked as private.
Agreed, I made it a private for `ModerationEventsQueryBuilder` and `ProductionLocationsQueryBuilder` separately.


> It should be an abstract method and defined in the OpenSearchQueryBuilderInterface, because the implementation of the _build_country is different in ModerationEventsQueryBuilder and ProductionLocationsQueryBuilder.

In Python, abstract base classes (ABCs) can contain private methods, but generally, private methods aren't declared abstract because the purpose of an abstract method is to enforce an interface on subclasses.